index.html

```diff
@@ -143,39 +152,41 @@ <h3 class="connect"><a href="">LET’S CONNECT</a></h3>
 				<li><a href=""><img src="img/chill.svg" alt=""></a></li>
 			</ul>
 			<button class="btn">Get my resume</button>
-	</div>
-	<div class="lang1">
-		<h3>Languages</h3>
-		<img src="/img/Union.svg" alt="">
-	</div>
-	<ul class="drop1">
-		<li>
-			<img src="/img/js_Ellipse 1.svg" alt="">
-			<p>Javascript</p>
-		</li>
-		<li>
-			<img src="/img/html_Ellipse 1.svg" alt="">
-			<p>HTML</p>
-		</li>
-		<li>
-			<img src="/img/css-Ellipse 1.svg" alt="">
-			<p>css</p>
-		</li>
-	</ul>
-	<div class="lang2">
-		<h3>Frameworks</h3>
-		<img src="/img/Unionarrdwn.svg" alt="">
-	</div>
-	<hr>
-	<div class="lang3">
-		<h3>skills</h3>
-		<img src="/img/Unionarrdwn.svg" alt="">
-	</div>
-	<hr>
+	    </div>
+		<div class="about_info">
+			<div class="lang1">
+				<h3>Languages</h3>
+				<img src="/img/Union.svg" alt="">
+			</div>
+			<ul class="drop1">
+				<li>
+					<img src="/img/js_Ellipse 1.svg" alt="">
+					<p>Javascript</p>
+				</li>
+				<li>
+					<img src="/img/html_Ellipse 1.svg" alt="">
+					<p>HTML</p>
+				</li>
+				<li>
+					<img src="/img/css-Ellipse 1.svg" alt="">
+					<p>css</p>
+				</li>
+			</ul>
+			<div class="lang2">
+				<h3>Frameworks</h3>
+				<img src="/img/Unionarrdwn.svg" alt="">
+			</div>
+			<hr>
+			<div class="lang3">
+				<h3>skills</h3>
+				<img src="/img/Unionarrdwn.svg" alt="">
+			</div>
+			<hr>
+		</div>	
 
 	</section>
-
-	<section class="contact" id="contact">
+	<section class="contact_info">
+	<div class="contact" id="contact">
 		<div class="contact_wrapper">
 			<h2 class="section_header">
 				Contact me
```

Comment:
- [ ] While you did a wonderful job in the last section of your project I've noticed that the `<input>` and `<textarea>` elements are not similar to the Figma template. They have implemented a border radius instead  be totally straight
- [ ] Finally, kindly set the "Get in Touch" button in the middle of your form to match to the Figma template :pray: 

**Your design**

![Screenshot from 2023-01-19 20-43-49](https://user-images.githubusercontent.com/105117832/213604003-88015df1-78b5-4968-a75b-8e24ca8869be.png)

**Figma design**

![Screenshot from 2023-01-19 20-44-00](https://user-images.githubusercontent.com/105117832/213604042-4424e151-32a1-4c20-8a3e-01cb50e53112.png)